examples/acephei/start.js

```diff
@@ -0,0 +1,8 @@
+require("accounts");
+require("books");
+require("products");
+require("reviews");
+
+setTimeout(() => {
+  require("gateway");
+}, 2);
```

Comment:
I removed the timeout altogether, this was a simple hack for us to load a client application on the side to create some load. Since we removed the client application to simplify things, I think it makes sense to simplify the script.